src/routes/PostButton.svelte

```diff
@@ -32,6 +33,12 @@
 	};
 
 	async function handlePost() {
+
+		if (lynt.length == 0) {
```

Comment:
What if the lynt is just whitespace?

Reply:
I don't think I was the one that added that I think I merged it from upstream (this repo)

Reply:
I added that but i didn't think of filtering out just whitespaces.